TransifexNativeSDK/clitool/src/main/java/com/transifex/clitool/MainClass.java

```diff
@@ -304,6 +305,12 @@ public Integer call() throws Exception {
                         "source locale can also be included.", paramLabel = "<locale>")
         String[] translatedLocales;
 
+        @Option(names = {"--fetch-tags"}, arity = "0..",
```

Comment:
Let's use `--with-tags-only` for consistency, as defined in the Python/Django [push command](https://github.com/transifex/transifex-python/blob/devel/transifex/native/django/management/utils/push.py#L44-L45).

Reply:
Done!